net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml

```diff
@@ -142,6 +142,18 @@
         <label>Elasticsearch URL</label>
         <type>text</type>
         <help>Set the URL where metrics shoud be sent to.</help>
+    </field>
+        <field>
+        <id>output.elastic_username</id>
+        <label>Elasticsearch username</label>
```

Comment:
Can you start usename and password upper case as we always do in labeling?

Reply:
Aha. Check now. In b6a0b85.